Tools-Override/tests.targets

```diff
@@ -27,7 +26,7 @@
     <UnsupportedPlatformsItems Include="$(UnsupportedPlatforms)"/>
   </ItemGroup>
 
-  <PropertyGroup Condition="'$(RunningOnNetFx)' != 'true'">
+  <PropertyGroup Condition="'$(BuildingNETFxVertical)' != 'true'">
```

Comment:
@weshaggard isn't these targets get used only when running the test? how would that can conflict with each other? 

anyway, @ericstj could you please advise if this needed to change it back to _bc_TargetGroup?